openpype/hosts/houdini/plugins/publish/collect_instances.py

```diff
@@ -109,6 +109,6 @@ def get_frame_data(self, node):
 
         data["frameStart"] = node.evalParm("f1")
         data["frameEnd"] = node.evalParm("f2")
-        data["steps"] = node.evalParm("f3")
```

Comment:
The old `steps` didn't work with the submit to deadline logic so I updated it. I also searched the code base but `"steps"` was used nowhere related to this so it seems the old code was just invalid.